2018/11/26 11:11:02 [error] 20165#20165: *2 connect() failed (111: Connection refused) while connecting to upstream, client: 116.24.67.45, server: 129.204.28.32, request: "GET /blog/ HTTP/1.1", upstream: "uwsgi://127.0.0.1:9001", host: "129.204.28.32:8000"
2018/11/26 11:12:08 [error] 20165#20165: *8 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 116.24.67.45, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/"
2018/11/26 11:12:34 [error] 20165#20165: *8 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 116.24.67.45, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/"
2018/11/26 11:13:44 [error] 20165#20165: *31 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 61.129.7.217, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/?nsukey=xs2BgTrd2B49wf3v0JUtvfoTS6gtjk1E0wGMIbN4J8BWKYZWaduNlihkI35B8fgdmHqoHyD0OmrZsA2Fs8sK0UrxG0IOr2QbxzSxwrZEdeZ7n6ujuZy7UsK3d99rnAJXjLlwee2DSOmhjOlJ4lKPVrG0heJSyAxRXh6ac4AH2H8wA9crhf2fvUoyp12FuRVKo2maR5F2F0FcVzsS2sondInxSZw3D3D"
2018/11/26 11:13:48 [error] 20165#20165: *38 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 61.151.180.39, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/?nsukey=xs2BgTrd2B49wf3v0JUtvfoTS6gtjk1E0wGMIbN4J8BWKYZWaduNlihkI35B8fgdmHqoHyD0OmrZsA2Fs8sK0UrxG0IOr2QbxzSxwrZEdeZ7n6ujuZy7UsK3d99rnAJXjLlwee2DSOmhjOlJ4lKPVrG0heJSyAxRXh6ac4AH2H8wA9crhf2fvUoyp12FuRVKo2maR5F2F0FcVzsS2sondInxSZw3D3D"
2018/11/26 11:13:49 [error] 20165#20165: *49 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 14.116.137.168, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/?nsukey=xs2BgTrd2B49wf3v0JUtvfoTS6gtjk1E0wGMIbN4J8BWKYZWaduNlihkI35B8fgdmHqoHyD0OmrZsA2Fs8sK0UrxG0IOr2QbxzSxwrZEdeZ7n6ujuZy7UsK3d99rnAJXjLlwee2DSOmhjOlJ4lKPVrG0heJSyAxRXh6ac4AH2H8wA9crhf2fvUoyp12FuRVKo2maR5F2F0FcVzsS2sondInxSZw3D3D"
2018/11/26 11:17:21 [error] 20165#20165: *66 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 116.24.67.45, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/aboutMe"
2018/11/26 11:17:22 [error] 20165#20165: *66 open() "/srv/Django/testWeb/blog/static/images/pattern.png" failed (2: No such file or directory), client: 116.24.67.45, server: 129.204.28.32, request: "GET /static/images/pattern.png HTTP/1.1", host: "129.204.28.32:8000", referrer: "http://129.204.28.32:8000/blog/"
